infra/dev/scripts/setup-postgres-linux.sh

```diff
@@ -54,7 +54,7 @@ TARGETARCH=$(dpkg --print-architecture)
 # Install PostgresSQL
 echo_header $GREEN "Step [1/4]: Installing PostgreSQL..."
 sudo apt update -y || handle_error "Failed to update package list."
-sudo apt install -y postgresql-$PG_MAIN_VERSION postgresql-contrib || handle_error "Failed to install PostgreSQL."su
+sudo apt install -y postgresql postgresql-contrib || handle_error "Failed to install PostgreSQL."
```

Comment:
Sure, I'll test it tomorrow

Reply:
I have actually merged more stuff, I think it also solves the -i issue

Reply:
@charlesBochet I just tested latest merged script and it seems working.

However, the `init.sql` file's location is incorrect in step 4. 
https://github.com/twentyhq/twenty/blob/main/infra/dev/scripts/setup-postgres-linux.sh#L78

```
=======================================================
Step [4/4]: Setting up database...
=======================================================
cp: cannot stat './postgres/init.sql': No such file or directory
```

It should be `./infra/dev/postgres/init.sql` instead of just `./postgres/init.sql`

Reply:
How are you running the script? Are you using make or the script directly.
It seems that the make root is ./infra/dev. That's why I've changed it as we will push people to use make

Reply:
I am executing the script directly (just following docs steps).

```
cd twenty
bash ./infra/dev/scripts/setup-postgres-linux.sh
```

Reply:
yep, good point, we need to update the doc to do: 
`make provision-postgres-linux`
I forgot to update it there